### PR TITLE
Update SMART Health IT section with updated instructions

### PR DESCRIPTION
**Pull request by kolkheang**
Addressing issue https://github.com/cerner/smart-on-fhir-tutorial/issues/48.  SMART Health IT site updated their Sandbox with the new [SMART Launcher](http://launch.smarthealthit.org/).  This PR is to update the tutorial to reflect this new change.  SMART Health IT site doesn't validate the `client_id` so this can be set to any value.

@zplata 
@kpshek 
@mjhenkes

### CODE REVIEW

**Comment by kolkheang** [comment]
I'll need to do additional work to re-organize the structure and enable the app to be framed by SMART Health IT Simulated EHR launch.  Our xfc framework is automatically setting the hidden flag when the app detects that it's being framed.

File: source/example-smart-app/launch-smart-sandbox.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <script src='./lib/js/fhir-client-cerner-additions-1.0.0.js'></script>
     <script>
       FHIR.oauth2.authorize({
-        'client_id': 'YOUR-SMART-HEALTH-IT-CLIENT-ID-HERE',
+        'client_id': '20a81fa7-8662-4ac7-afbf-bcfb167c07cf',
```
[the remark this answers is not in the document]
Comment (kpshek) [reply source/example-smart-app/launch-smart-sandbox.html]: I don't think we want to put a default `client_id` here.

[the remark this answers is not in the document]
Comment (kpshek) [reply source/example-smart-app/launch-smart-sandbox.html]: Hmm, I understand better now after running through the new SMART Health IT launcher. They don't care about the `client_id` value at all.

> The app's client_id is not validated on the SMART test server, so any text string will work.

While that works for the SMART Health IT launcher, we still want them to specify their own `client_id` they obtained by registered in the code Console, right?

Also, perhaps we should put a comment in the code (and potentially in the instructions as well) that the SMART Health IT launcher ignores the `client_id` value.

[the remark this answers is not in the document]
Comment (jmandel) [reply source/example-smart-app/launch-smart-sandbox.html]: (We'd prefer to revert this line of the commit so it just  says `'client_id': 'YOUR-SMART-HEALTH-IT-CLIENT-ID-HERE',`.)

**Pull request merged**